dev/tasks/linux-packages/apache-arrow/apt/ubuntu-focal/Dockerfile

```diff
@@ -79,3 +80,9 @@ RUN \
   ln -fs /usr/local/bin/gi-docgen /usr/bin && \
   apt clean && \
   rm -rf /var/lib/apt/lists/*
+
+ARG cmake=3.25.0
+RUN curl -L \
+    "https://github.com/Kitware/CMake/releases/download/v${cmake}/cmake-${cmake}-linux-$(uname -m).tar.gz" | \
+    tar -xzf - --directory /usr/local --strip-components=1 && \
+    ln -fs /usr/local/bin/cmake /usr/bin/cmake
```

Comment:
Is the last (`ln -fs`) actually required? Usually, `/usr/local/bin` takes precedence over `/usr/bin`.

Reply:
Let's try this.

Reply:
Reverted back and added a comment on why the link is required:
https://github.com/apache/arrow/pull/44989/commits/2da76a52dad4ed0a68d7ec1be149e04880c4da5e
We still have to create the link on debian rules or it fails to find cmake